controllers/nginx/configuration.md

```diff
@@ -51,6 +51,9 @@ The following annotations are supported:
 |[ingress.kubernetes.io/upstream-max-fails](#custom-nginx-upstream-checks)|number|
 |[ingress.kubernetes.io/upstream-fail-timeout](#custom-nginx-upstream-checks)|number|
 |[ingress.kubernetes.io/whitelist-source-range](#whitelist-source-range)|CIDR|
+|[ingress.kubernetes.io/sticky-enabled](#sticky-session)|true or false|
+|[ingress.kubernetes.io/sticky-name](#sticky-session)|string|
+|[ingress.kubernetes.io/sticky-hash](#sticky-session)|string|
 
```

Comment:
@fanhaozzu Agreed. I just think that by now we could merge this, and add this feature later (we can open a new issue for this one).

This is because this item would need some more work to validate it's input, as this is a time pattern. Maybe using [this](https://golang.org/pkg/time/#ParseDuration)

Can we work in this item latelly?